src/main/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/impl/EmployeesTransformerImpl.java

```diff
@@ -29,19 +29,56 @@ public Employees getEmployees(EmployeesApi employeesApi) {
       
       return employees;
     }
-    
-    private void populateCurrentPeriodForWeb(EmployeesApi employeesApi, Employees employees, AverageNumberOfEmployees averageNumberOfEmployees) {     
-        CurrentPeriod currentPeriod = employeesApi.getCurrentPeriod();  
+
+    @Override
+    public EmployeesApi getEmployeesApi(Employees employees) {
+
+        EmployeesApi employeesApi = new EmployeesApi();
+
+        setCurrentPeriodEmployeesOnApiModel(employees, employeesApi);
+        setPreviousPeriodEmployeesOnApiModel(employees, employeesApi);
+
+        return employeesApi;
+    }
+
+    private void populateCurrentPeriodForWeb(EmployeesApi employeesApi, Employees employees, AverageNumberOfEmployees averageNumberOfEmployees) {
+        CurrentPeriod currentPeriod = employeesApi.getCurrentPeriod();
         if (currentPeriod != null) {
             employees.setDetails(currentPeriod.getDetails());
             averageNumberOfEmployees.setCurrentAverageNumberOfEmployees(currentPeriod.getAverageNumberOfEmployees());
         }
     }
-    
-    private void populatePreviousPeriodForWeb(EmployeesApi employeesApi, AverageNumberOfEmployees averageNumberOfEmployees) {     
-        PreviousPeriod previousPeriod = employeesApi.getPreviousPeriod();  
+
+    private void populatePreviousPeriodForWeb(EmployeesApi employeesApi, AverageNumberOfEmployees averageNumberOfEmployees) {
+        PreviousPeriod previousPeriod = employeesApi.getPreviousPeriod();
         if (previousPeriod != null) {
             averageNumberOfEmployees.setPreviousAverageNumberOfEmployees(previousPeriod.getAverageNumberOfEmployees());
         }
-    }    
+    }
+
+    private void setCurrentPeriodEmployeesOnApiModel(Employees employees, EmployeesApi employeesApi) {
+        CurrentPeriod currentPeriod = new CurrentPeriod();
+
+        if (employees.getDetails() != null && employees.getDetails().equals("")) {
```

Comment:
Only minor, but could use `StringUtils.isBlank` here for readability

Reply:
Agreed.